cocos/core/pipeline/render-queue.ts

```diff
@@ -74,6 +75,7 @@ export class RenderQueue {
     constructor (desc: IRenderQueueDesc) {
         this._passDesc = desc;
         this._passPool = new RecyclePool<IRenderPass>(() => ({
+            sortingPriority: SortingManager.getSortingPriority(),
```

Comment:
It shouldn't get sorting priority here, just use a default value and make sure it will updated